TCMManagement/Controllers/AppointmentController.cs

```diff
@@ -1,38 +1,39 @@
-using System.Collections.Generic;
-using System.Linq;
-using System.Web.Http;
-using System.Net.Http;
-using TCMManagement.BusinessLayer;
-using TCMManagement.Models;
-using static TCMManagement.BusinessLayer.Constants;
-
-namespace TCMManagement.Controllers
-{
-    /// <summary>
-    /// All Controllers should follow CRUD sequence so it will be easier to maintain the code.
-    /// </summary>
-    public class AppointmentController : ApiController
-    {
-        private IEntityServices<Appointment> appointmentService;
-
-        public AppointmentController() {
-            appointmentService = new AppointmentService();
-        }
-
-        public AppointmentController(IEntityServices<Appointment> service)
-        {
-            appointmentService = service;
-        }
-
-        // Comment our for now, easier to test
-        // [Authorize(Roles = "Receptionist")]
-        [HttpPost]
-        public IHttpActionResult AddAppointment(Appointment a)
-        {
-            return Ok(appointmentService.CreateItem(a));
-        }
-
-        // querystring = "?person=1"
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+using System.Net.Http;
+using TCMManagement.BusinessLayer;
+using TCMManagement.Models;
+using static TCMManagement.BusinessLayer.Constants;
+
+namespace TCMManagement.Controllers
+{
+    /// <summary>
+    /// All Controllers should follow CRUD sequence so it will be easier to maintain the code.
+    /// </summary>
+    public class AppointmentController : ApiController
+    {
+        private IEntityServices<Appointment> appointmentService;
+
+        public AppointmentController() {

```

Comment:
This constructor can be removed since DI is used.

Reply:
OK, will remove. Sorry, what is DI ?

Reply:
Dependency Injection.
Which means that Autofac will handle the creation service passed into this constructor automatically.

Reply:
Ok, got it. Thanks!